google/cloud/documentai_toolbox/wrappers/entity.py

```diff
@@ -95,9 +95,11 @@ def crop_image(
         if not documentai_page.image:
             raise ValueError("Document does not contain images.")
 
-        top, left, bottom, right = docai_utilities.get_bounding_box(
+        bbox = docai_utilities.get_bounding_box(
             bounding_poly=self.documentai_object.page_anchor.page_refs[0].bounding_poly,
             page_dimension=documentai_page.dimension,
         )
+        if not bbox:
```

Comment:
In this case let's use `if bbox is None`.  Also perhaps move this check to immediately after we get `bbox`.

Reply:
Done, although I'm not sure of your second comment. This is already right after we get bbox.